modules/installation/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -1,6 +1,6 @@
 [id='proc-rhdh-deploy-eks-using-operator_{context}']
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+= Installing {product} on {eks-name} ({eks-short}) using the Operator
 
 You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
```

Comment:
based on input from Tim in RHIDP-2707, we don't want to encourage/support opertor on k8s without first installing OLM as a prereq.